keda/templates/22-metrics-deployment.yaml

```diff
@@ -23,10 +23,10 @@ spec:
         aadpodidbinding: {{ .Values.podIdentity.activeDirectory.identity }}
         {{- end }}
       annotations:
-      {{- if .Values.prometheus.enabled }}
+      {{- if .Values.prometheus.metricServer.enabled }}
```

Comment:
This is a breaking change. Do we change it or ship a new major version @zroubalik? Maybe we can be backwards compatible and track it in an issue to delete it for v3.0?

Reply:
Thoughts on this @zroubalik ?

Reply:
By v3.0 you mean KEDA v3.0? Honestly, I prefer to not doing another major version bump in a near future 😄 

I think we can introduce this breaking change, as prometheus metrics are still quite new, so if we change it soon, the change itself could potentially hit just a few users.

Reply:
Let's make this an exception then, you owe me a coca cola 😂